index.js

```diff
@@ -1,7 +1,18 @@
-var hat = require('hat');
+'use strict';
+
+var cuid = require('cuid');
 
 function convert(geojson) {
-    var sourceId = hat();
+    var sourceId = cuid();
+
+    // If a LineString has fill properties, we need to add a source layer for it
```

Comment:
A “source layer” is a concept found only in vector sources in style JSON. Do you mean “fill layer”?